content/docs/resiliency/_index.md

```diff
@@ -48,9 +48,9 @@ CSM for Resiliency provides the following capabilities:
 ## Supported Storage Platforms
 
 {{<table "table table-striped table-bordered table-sm">}}
-|               | PowerFlex    | Unity XT                   |
-| ------------- | :----------: | :------------------------: |
-| Storage Array | 3.5.x, 3.6.x | 5.0.5, 5.0.6, 5.0.7, 5.1.0, 5.1.2 |
+|               | PowerFlex    | Unity XT                          | PowerScale     |
+| ------------- | :----------: | :-------------------------------: | :------------: |
+| Storage Array | 3.5.x, 3.6.x | 5.0.5, 5.0.6, 5.0.7, 5.1.0, 5.1.2 | OneFS 9.2, 9.3 |
 {{</table>}}
 
```

Comment:
Please also update Kubernetes to include 1.21 - 1.24
And OpensShift 4.9/ 4.10

Reply:
Updated

Reply:
PowerScale supports :- 
OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4

Reply:
I believe we need to remove Open Shift 4.8 support. Drivers support only Open Shift 4.9,4.10 and 4.10EUS for CSM 1.3

Reply:
Fixed